### PR TITLE
feat : update studio docs

### PR DESCRIPTION
**Pull request by Ayush0054**
## Description

Describe key changes, mention related issues or motivation for the changes.

**Note:** Your PR title must follow conventional commit format (e.g., `docs: add auth guide`, `fix: correct broken links`, `style: update formatting`). See [CONTRIBUTING.md](../CONTRIBUTING.md) for details.

## Type of Change

- [ ] Bug fix (errors, broken links, outdated info)
- [ ] New content
- [ ] Content improvement
- [ ] Other: \_\_\_\_

## Related Issues/PRs (if applicable)

- Closes #\_\_\_\_
- Related SDK PR: agno-agi/agno#\_\_\_\_

## Checklist

- [ ] Content is accurate and up-to-date
- [ ] All links tested and working
- [ ] Code examples verified (if applicable)
- [ ] Spelling and grammar checked
- [ ] Screenshots updated (if applicable)

### CODE REVIEW

File: agent-os/studio/agents.mdx [hunk 1]
```diff
@@ -3,21 +3,28 @@ title: "Agents"
...
 
-| Setting | Description |
-|---------|-------------|
-| **Model** | Select from registered models |
-| **Tools** | Attach registered tools and toolkits |
-| **Instructions** | System-level instructions for the agent |
-| **Input/Output Schema** | Structured I/O using registered Pydantic schemas |
-| **Memory** | Enable memory for multi-turn conversations |
-| **Knowledge** | Attach knowledge bases for RAG |
+- **Model**: select from registered models.
```
Comment (mishramonalisha76) [inline agent-os/studio/agents.mdx:12]: Should we also give the sdk version from when studio is available ?

File: agent-os/studio/introduction.mdx [hunk 2]
```diff
@@ -3,25 +3,36 @@ title: "Overview"
...
 
 ## Concepts
 
-| Concept                                 | Description                                                                         |
-| --------------------------------------- | ----------------------------------------------------------------------------------- |
-| [Agents](/agent-os/studio/agents)       | Build and configure agents with models, tools, and instructions                     |
-| [Teams](/agent-os/studio/teams)         | Compose multi-agent teams with coordination modes                                   |
-| [Workflows](/agent-os/studio/workflows) | Design step-based workflows with conditions, loops, routers, and parallel execution |
-| [Registry](/agent-os/studio/registry)   | Browse and manage registered tools, models, databases, and schemas                  |
+**[Agents](/agent-os/studio/agents)**
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]:
```suggestion
**[Agents](/agent-os/studio/agents)** :
```

File: agent-os/studio/introduction.mdx [hunk 3]
```diff
@@ -3,25 +3,36 @@ title: "Overview"
...
+Build agent by giving it a model, tools, and instructions. Add knowledge and memory to ground its responses and remember context.
+
+**[Teams](/agent-os/studio/teams)**
+Build multi-agents team that works toward a shared goal. Choose how the leader coordinates with members using `coordinate`, `route`, `broadcast`, or `tasks` mode.
+
+**[Workflows](/agent-os/studio/workflows)**
+Orchestrate agents and teams into step-based pipelines. Control the flow with loops, conditions, routers, and parallel execution.
+
+use functions or cel expressions to evaluate conditions and selectors.
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]: Does this belong to workflows only or agent and teams too ? looks like an orphaned line

File: agent-os/studio/introduction.mdx [hunk 4]
```diff
@@ -3,25 +3,36 @@ title: "Overview"
...
+
+**[Registry](/agent-os/studio/registry)**
+Add tools, models, databases, and schemas that Studio can use to build agents, teams, and workflows.
+
+Register agents and teams in the `Registry` to reuse them as members in Studio teams and as steps in Studio workflows.
+
+Register `knowledge`, `memory_managers`, and `session_summary_managers` for agents and teams to use.
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]:
```suggestion
Register the components Studio composes with: tools, models, databases, schemas, knowledge, `memory_managers`, and `session_summary_managers`. Register existing agents and teams to reuse them as members in Studio teams or steps in Studio workflows.
```

File: agent-os/studio/introduction.mdx [hunk 5]
```diff
@@ -54,38 +65,70 @@ Studio manages the full development lifecycle from building to deploying complex
...
+  <img
+    src="/images/agent-os-studio-create.png"
+    alt="Create an agent in Studio"
+  />
 
-### 2. Save Draft
 
-Save your work as a draft version. Drafts are not accessible via the API but can be tested, restored, and published later. You can save multiple draft versions to checkpoint your progress.
+
+### 2. Save Draft or publish directly
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]:
```suggestion
### 2. Save Draft or Publish Directly
```

File: agent-os/studio/introduction.mdx [hunk 6]
```diff
@@ -54,38 +65,70 @@ Studio manages the full development lifecycle from building to deploying complex
...
+
+- Save your work as a draft or publish directly. Drafts can be edited and updated. Drafts helps you test your component before publishing it.
+- Publish or Draft multiple versions to checkpoint your progress.
+  <img
+    src="/images/agent-os-studio-draft-publish-btn.png"
+    alt="Save draft or publish directly in Studio"
+    style={{ borderRadius: "8px" }}
+  />
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]: We don't need to show this button image, it doesnot give any idea . we can skip an image here

File: agent-os/studio/introduction.mdx [hunk 7]
```diff
@@ -54,38 +65,70 @@ Studio manages the full development lifecycle from building to deploying complex
...
 
 ### 3. Test
 
-Once saved, test your draft in the AgentOS Control Plane:
+Once saved, test your draft or published version in the AgentOS:
+
+- **Chat Page**: 
+  - Interact with your agent, team, or workflow in real-time
+  - run specific version of the component by selecting it in the dropdown.
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]:
```suggestion
  - Run specific version of the component by selecting it in the dropdown.
```

File: agent-os/studio/introduction.mdx [hunk 8]
```diff
@@ -54,38 +65,70 @@ Studio manages the full development lifecycle from building to deploying complex
...
 - **Debug Mode**: Enable verbose logging to troubleshoot issues
 
+
 <Warning>
   Before publishing, test and make sure your agent handles edge cases and
   unexpected inputs gracefully.
 </Warning>
 
-### 4. Publish
+### 4. Delete
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]: Don't think we should include delete as a seperate point, it  can be a part inside Manage versions? or skip it entirely

File: agent-os/studio/introduction.mdx [hunk 9]
```diff
@@ -54,38 +65,70 @@ Studio manages the full development lifecycle from building to deploying complex
...
+  style={{ borderRadius: "16px" }}
+/>
+- **Restore**: edit and update the draft version.
 - **Set Current**: Choose which published version is used by default when running via the API
-- **Delete**: Remove old versions you no longer need
+
+- only draft versions can be updated and edited in the Studio.
+- published versions are immutable and can only be updated by publishing a new version.
+- draft version cannot be set as current.
+- only draft versions can be deleted from versions page , published versions and current config cannot be deleted.
```
Comment (mishramonalisha76) [inline agent-os/studio/introduction.mdx]: Lets start each sentence with a capital character

Comment (mishramonalisha76) [reply agent-os/studio/introduction.mdx]:
```suggestion
Two actions are available on this page:

- **Restore**: open a draft for editing in Studio.
- **Set Current**: choose which published version the API serves.

Drafts are mutable: you can edit them in Studio and delete them. Published versions are immutable and undeletable. To change a published version, publish a new one. Only published versions can be set as current.
```

File: agent-os/studio/agents.mdx [hunk 10]
```diff
@@ -3,21 +3,28 @@ title: "Agents"
 description: "Build and configure agents visually in AgentOS Studio."
 ---
 
-Build production-grade agents via AgentOS Studio’s visual canvas. Wire up models, tools, 
-and structured I/O into complex agentic workflows. Deploy instantly. No-code required.
+Build production-grade agents via AgentOS Studio’s. Wire up models, tools, 
+instructions, knowledge, memory, and more. Deploy instantly. No-code required.
```
Comment (mishramonalisha76) [inline agent-os/studio/agents.mdx]:
```suggestion
Build agents in AgentOS Studio by wiring up models, tools, instructions, knowledge, and memory. No code required.
```

File: agent-os/studio/teams.mdx
Comment (mishramonalisha76) [inline agent-os/studio/teams.mdx]: Team is not visual builder as of now right ?

File: agent-os/studio/teams.mdx [hunk 11]
```diff
@@ -9,14 +9,33 @@ No code required. Drag agents onto the canvas, configure coordination, and run t
...
-
-| Setting | Description |
-|---------|-------------|
-| **Members** | Drag and drop agents to include in the team |
-| **Mode** | Coordination mode: `coordinate`, `route`, or `collaborate` |
-| **Instructions** | Team-level instructions for the leader agent |
-| **Success Criteria** | Define when the team's task is complete |
+Create a new team by using existing agents and teams from your Registry or built in Studio, and configuring the team settings:
+
+- **Team Members and Execution**: select agents or teams to include as members, and set the team mode and delegation behavior. read more about [team members and execution](/agent-os/studio/teams#team-members-and-execution) here.
```
Comment (mishramonalisha76) [inline agent-os/studio/teams.mdx]:
```suggestion
- **Team Members and Execution**: select agents or teams to include as members, and set the team mode and delegation behavior. Read more about [team members and execution](/agent-os/studio/teams#team-members-and-execution) here.
```

File: agent-os/studio/teams.mdx [hunk 12]
```diff
@@ -9,14 +9,33 @@ No code required. Drag agents onto the canvas, configure coordination, and run t
...
+<img
+  src="/images/agent-os-studio-create-team.png"
+  alt="Create a team in AgentOS Studio"
+  style={{ borderRadius: "8px" }}
+/>
+
+### Team Members and Execution
+
+- **Members**: select the agents or teams to include as members.
+- **Team Mode** _(optional)_: controls how the team leader coordinates work with member agents:
```
Comment (mishramonalisha76) [inline agent-os/studio/teams.mdx]: We should attach  some reference  doc for  this if available

File: agent-os/studio/teams.mdx [hunk 13]
```diff
@@ -9,14 +9,33 @@ No code required. Drag agents onto the canvas, configure coordination, and run t
...
+  - **None**
+  - **Coordinate** (default)
+  - **Route**
+  - **Broadcast**
+  - **Tasks**
+- **Respond Directly**: let the team leader respond on its own instead of delegating.
```
Comment (mishramonalisha76) [inline agent-os/studio/teams.mdx]:
```suggestion
- **Respond Directly**: lets the team leader respond on its own instead of delegating.
```

File: agent-os/studio/teams.mdx [hunk 14]
```diff
@@ -9,14 +9,33 @@ No code required. Drag agents onto the canvas, configure coordination, and run t
...
+- **Delegate to All Members**: send the task to every member at once.
```
Comment (mishramonalisha76) [inline agent-os/studio/teams.mdx]:
```suggestion
- **Delegate to All Members**: sends the task to every member at once.
```

File: agent-os/studio/registry.mdx [hunk 15]
```diff
@@ -4,7 +4,33 @@ sidebarTitle: "Registry"
...
+
+<img
+  src="/images/agent-os-studio-registry.png"
+  alt="Open Studio Registry from the Studio navigation"
+  style={{ borderRadius: "8px" }}
+/>
+
+
+
+example of registry configuration:
```
Comment (mishramonalisha76) [inline agent-os/studio/registry.mdx]:
```suggestion
Example of registry configuration:
```

**Review by mishramonalisha76 — COMMENTED** [review]
Can we add a small video of creating agent, team and workflows , individual pages or maybe one in the overview ?

Can we make all the image to be of the whole app witht he sidebar and topbar?

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
Lets also connect each page to the respective API reference

**Comment by Ayush0054** [comment]
in overview i will add this up as a follow-up
 > Can we add a small video of creating agent, team and workflows , individual pages or maybe one in the overview ?


i dont think we need image with sidebar and topbar , since it gets very hard to show without zooming
 > Can we make all the image to be of the whole app witht he sidebar and topbar?

**Review by mishramonalisha76 — DISMISSED** [review]
We can add the video from there http://localhost:3001/agent-os/control-plane ?

**Comment by Ayush0054** [comment]
> We can add the video from there http://localhost:3001/agent-os/control-plane ?

done

**Review by mishramonalisha76 — APPROVED** [review]
